packages/email/email_tests.js

```diff
@@ -83,7 +83,7 @@ Tinytest.add("email - multiple e-mails same stream", function (test) {
       text: "This is the body\nof the message\nFrom us.",
     });
 
-    var contents;
+    let contents;
```

Comment:
why not const? We don't need to reuse this variable and change its value

Reply:
Fixed!